### PR TITLE
Disable inbound connections on mobiles

### PR DESCRIPTION
**Pull request by dshulyak**
Since we are not using p2p server v4 discovery and dialing i had to add another control option to p2p config

Closes: https://github.com/status-im/status-go/issues/978

### CODE REVIEW

File: geth/node/node_test.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+				key, _ := crypto.GenerateKey()
+				peer := &p2p.Server{
+					Config: p2p.Config{
+						NoDiscovery: true,
+						PrivateKey:  key,
+						MaxPeers:    1,
+						ListenAddr:  "127.0.0.1:0",
+					},
+				}
+				assert.NoError(t, peer.Start())
```
Comment (adambabik) [inline geth/node/node_test.go]: Not sure if this `assert` makes sense. If the p2p server fails to start, it's a problem with configuration or system resources. What's the advantage of continuing with the rest of the test?

File: geth/node/node_test.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+			percent: 70,
+		},
+		{
+			name:    "None",
+			max:     4,
+			percent: 0,
+		},
+	} {
+		t.Run(tc.name, func(t *testing.T) {
+			key, _ := crypto.GenerateKey()
```
Comment (adambabik) [inline geth/node/node_test.go]: Maybe we can try `t.Parallel()`?

**Review by adambabik — APPROVED** [review]
LGTM provided that CI passes.

**Comment by adambabik** [comment]
What's the status of this PR?

**Comment by dshulyak** [comment]
need to push a couple of fixed suggestions and test it on mobile before merging

**Comment by dshulyak** [comment]
I will restore it later

**Pull request closed without merging**